wordcloud/wordcloud.py

```diff
@@ -503,16 +508,17 @@ def process_text(self, text):
 
         flags = (re.UNICODE if sys.version < '3' and type(text) is unicode
                  else 0)
-        regexp = self.regexp if self.regexp is not None else r"\w[\w']+"
+        regexp = self.regexp if self.regexp is not None else r"(\w[\w']+|\w+)"
```

Comment:
All right, I didn't notice that you really want to filter out that words. Can you give me other use cases for the matching desired? So far, that case of 1 char words can be avoided with a filter using the condition `len(word)==1 and not word.isdigit()`, but if you're thinking in more cases then it's not worth to "hard code" the filtering process.